internal/features/features_test.go

```diff
@@ -0,0 +1,43 @@
+package features_test
+
+import (
+	"os"
+	"runtime"
+	"testing"
+
+	"github.com/tetratelabs/wazero/internal/features"
+	"github.com/tetratelabs/wazero/internal/testing/require"
+)
+
+func init() {
+	os.Setenv(features.EnvVarName, "f0,f1,f2")
+}
+
+func TestList(t *testing.T) {
+	require.Equal(t, []string{"f0", "f1", "f2"}, features.List())
+}
+
+func TestEnabled(t *testing.T) {
+	require.True(t, features.Enabled("f0"))
+	require.True(t, features.Enabled("f1"))
+	require.True(t, features.Enabled("f2"))
+	require.False(t, features.Enabled("nope"))
+}
+
+func TestAllocsEnabled(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("accessing features allocates memory on windows")
```

Comment:
ps I don't fully understand the issue here based on the comment. Why is allocating memory during a test a problem? Is it not de-allocated?

Reply:
I'll rephrase 👍